WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/OverviewMapper.kt

```diff
@@ -196,13 +196,35 @@ class OverviewMapper
         if (shouldShowVisitors) {
             result.add(ChartLegend(R.string.stats_visitors))
         }
+
+        val entryType = when (selectedType) {
+            1 -> R.string.stats_visitors
```

Comment:
Thanks! That sounds like a good idea! I did this here because the same approach was being used above to determine the `selectedType` so I thought doing this would make it easier to spot the relation.

Reply:
Yeah, I saw it and I understand your decision. However, I thought introducing an enum might still be worth it and would make the code clearer.